step.yml

```diff
@@ -2,31 +2,52 @@ title: "Deploy to Bitrise.io - Apps, Logs, Artifacts"
 
   ### Configuring the Step
 
-  1. Set the value for the Deploy directory of file path. The default is the $BITRISE_DEPLOY_DIR environment variable which is exposed by the Bitrise CLI.
-  2. Set the **Notify: User Roles** and the **Notify: Emails** inputs to configure notifications for the users. Note that the **Notify: Emails** input is only used if a public install page is created and you must use a Secret to set it.
-  3. Optionally, you can create a zip file for all your build artifacts by setting the **Compress the artifacts into one file?** input to `true`. 
-
-  Get a public install page that allows testers to install the app on their devices:
-
-  1. Set the **Enable public page for the App?** input to `true`.
-  2. Set the **Compress the artifacts into one file?** input to `false`. If all the contents of the deploy directory are compressed into a single zip file, there won't be a public install page.
+  1. Set the value for the **Deploy directory or file path** required input. It can be a path to a directory or a file. If you provide a directory, everything in that directory gets uploaded recursively. 
+  If you provide only a file, then only that file gets uploaded. The default value is the `$BITRISE_DEPLOY_DIR` Env Var which is exposed by the Bitrise CLI.
+  2. Set the value of the **Notify: User Roles** input. It sends an email with the [public install URL](https://devcenter.bitrise.io/deploy/bitrise-app-deployment/) to those Bitrise users whose roles are included in this field. 
+  The default value is `everyone`. If you wish to notify based on user roles, add one or more roles and separate them with commas, for example, `developers`, `admins`. If you don't want to notify anyone, set the input to `none`.
+  3. Set the **Notify: Emails** sensitive input. It sends the public install URL in an email to the email addresses provided here. If you’re adding multiple email address, make sure to separate them with commas. 
+  The recipients do not have to be in your Bitrise team. Please note that if the email address is associated with a Bitrise account, the user must be [watching](https://devcenter.bitrise.io/builds/configuring-notifications/#watching-an-app) the app.
+  4. The **Enable public page for the App?** required input is set to `true` by default. It creates a long and random URL which can be shared with those who do not have a Bitrise account. 
+  If you set this input to `false`, the **Notify: Emails** input will be ignored and the **Notify: User Roles** will receive the build URL instead of the public install URL.
+  5. With the **Compress the artifacts into one file?** required input set to `true`, you can compress the artifacts found in the Deploy directory into a single file.
+  You can specify a custom name for the zip file with the `zip_name` option. Otherwise the default Deploy directory will be used. 
```

Comment:
`Otherwise the default Deploy directory will be used`

Maybe somthing like 

`Otherwise the default Deploy directory's name will be used`

would be better?